docs/guides/deploying_to_digital_ocean.md

```diff
@@ -0,0 +1,37 @@
+# Deploying with digital ocean
+
+There are multiple ways to deploy to digital ocean. We will deploy with docker swarm in this guide for simplicity.
+(note: while it's possible to use docker-compose it's generally not recommended for production. Since it's not much more difficult to use docker swarm, that's what we recommend.)
```

Comment:
I'm not sure what you mean in the last sentence here ("Since it's not much more difficult to use docker swarm, that's what we recommend"). Could you clarify what you mean?

Also, looking through the doc, you should make sure to capitalize all the pronouns (i.e. `digital ocean` should always be `Digital Ocean`, and `docker swarm` should always be `Docker Swarm`)

Reply:
I'll leave out the last sentence then, it might be easier.
I originally added that sentence for somebody like. Docker-compose was just so easy, and I thought docker-swarm would be a lot heavier (a bit like kubernetes I guess).

Reply:
Ah I see, I would maybe rephrase this paragraph to:

While there are multiple ways one can deploy to Digital Ocean, for this guide we're choosing to use Docker Swarm (installed as a one-click app). This choice was made because it is easy to get started with on Digital Ocean, and makes for a good foundation for small production applications due to it's convenience and feature set.